testeng/resources/cleanup-python-code.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+rm -rf code_cleanup_venv
+virtualenv --python=python"$PYTHON_VERSION" code_cleanup_venv -q
+source code_cleanup_venv/bin/activate
+
+echo "Upgrading pip..."
+pip install pip==20.0.2
+
+pip install -r cleanup-python-code-requirements.txt
+
+echo "Getting current sha..."
+cd "$REPO_NAME"
+export CURRENT_SHA=$(git rev-parse HEAD)
+
+echo "Running clean up scripts"
+black --line-length=120 .
+isort --recursive .
+2to3 -f asserts -f future -f except -f metaclass -w .
+
+find . -type f -name "*.py" | while read fname; do
```

Comment:
`2to3` and `pyupgrade` both create backup files with extension `.bak`, I think after all the processing, all such files should be deleted.

Reply:
done